PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -161,7 +162,7 @@ def miniAOD_customizeCommon(process):
     )
 
     addJetCollection(process, postfix   = "", labelName = 'Puppi', jetSource = cms.InputTag('ak4PFJetsPuppi'),
-                    jetCorrections = ('AK4PF', ['L1FastJet', 'L2Relative', 'L3Absolute'], ''),
+                    jetCorrections = ('AK4PFchs', ['L2Relative', 'L3Absolute'], ''),
                     algo= 'AK', rParam = 0.4, btagDiscriminators = map(lambda x: x.value() ,process.patJets.discriminatorSources)
                     )
     
```

Comment:
As @gpetruc said, it is needed if the intention is to store some of the SV info as userFloats. If not, then there is no need to add TagInfos. Also note that for slimmedJetsAK8 TopTagInfos are stored for later access to the top tagging information.